client/src/components/musicPlayer.jsx

```diff
@@ -1,24 +1,178 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
-import songs from '../../../sampleData.js';
+import styled from 'styled-components';
+
+
+const MusicPlayerContainer = styled.div`
+  display: flex;
+  flex-direction: row; 
+  justify-content: flex-end;
+  background: linear-gradient(135deg, #4D586E, #3A67C0);
+  height: 340px;
+  width: 1200px;
+  margin: 0 auto;
+  color: white;
+  padding-top: 15px;
+  padding-left: 25px;
+  padding-bottom: 15px;
+  padding-right: 15px;
+`;
+
+const ButtonTitleContainer = styled.div`
+  padding-top: 20px;
+  display: flex;
+  flex-direction: row; 
+`;
+
+const TitleArtistContainer = styled.div`
+  padding-left: 10px;
+  display: flex;
+  flex-direction: column; 
+  width: 400px;
+  font-weight: 100;
+`;
+
+const PlayButtonContainer = styled.div`
+  justify-content: flex-start;
+  border-radius: 50%;
+  border-color: #FF5836;
+  background: #FF5836;
+  width: 60px;
+  height: 60px;
+  cursor: pointer;
+  &:hover {
+    background: #FF4137;
+  }
+`;
+const PlayButton = styled.img`
+  margin-left: 7px;
+  margin-top: 4px;
+  width: 50px;
+  height: 50px;
+`;
+
+const ArtistContainer = styled.div`
+  
```

Comment:
are you intending to fill these in later?

Reply:
They're kind of on a need basis. In general they look more organized when I have <somethingContainer> in the render as opposed to unlabelled divs.